### ISSUE TITLE
Gui interface for contact

### ISSUE THREAD

**Issue by diveshuttam** [dashohoxha/egpg#80]
- [x] subtask  #86 (add contact and subparts)
- [x] export contact
- [x] certify
- [x] trust
- [x] delete

**Comment by diveshuttam** [comment]
I am currently working on this, it is a bit difficult interface so might be needed to made in python though I am trying if I can do it in yad only.
In any case I'll finish this soon in the best way it seems to me.

**Comment by dashohoxha** [comment]
Try to keep it simple and make it with `yad`. We can improve it later (with Python etc.) if needed.

Also, I don't know why you continued with this task. Fixing the initialization sequence, generating a new key, and managing the settings seem to be more important. But it is Ok.

Let me know if you have difficulties or need my help. Normally I am trying to work on some other things and I think that you are doing fine, but if you need help I will try to find time.

**Comment by diveshuttam** [comment]
> Also, I don't know why you continued with this task. 

I had continued with this as it is required for some other tasks like `seal` etc.
 
I have created a basic interface for contact and seal as well ( [refer](https://github.com/EasyGnuPG/egpg/tree/gui_contacts) ). I haven't tried it with the container yet hence not creating the PR. 

For making it work, I needed commands from newer version of yad (specifically for `--select-action=on_select` in contacts list). Though I have made it work, it has introduced some minor UI errors like small sized windows, some problems with text. So I'll create the PR after fixing them. 

(PS i am using yad 0.40.0 currently earlier it was 0.27.0)
